src/category_theory/limits/shapes/images.lean

```diff
@@ -405,4 +417,109 @@ def im : arrow C ⥤ C :=
 
 end has_image_maps
 
+section strong_epi_mono_factorisation
+
+/-- A strong epi-mono factorisation is a decomposition `f = e ≫ m` with `e` a strong epimorphism
+    and `m` a monomorphism. -/
+structure strong_epi_mono_factorisation {X Y : C} (f : X ⟶ Y) :=
+(I : C)
+(e : X ⟶ I)
+(m : I ⟶ Y)
+(fac : e ≫ m = f)
```

Comment:
If there's a reason to not `extend mono_factorisation`, you can still use `obviously` to help synthesize this proof. I agree it doesn't save much in this file, but it's pretty harmless to provide the convenience for later users.